packages/block-library/src/site-title/style.scss

```
@@ -1,3 +1,7 @@
+:root :where(.wp-block-site-title) {
```

```suggestion
.wp-block-site-title {
```
The `:root :where()` selector results in `0-1-0` specificity the same as `.wp-block-site-title` just using more characters and data. Perhaps we should just simplify that?

---

Sure

---

Sorry @akasunil, I meant only for the `box-sizing` rule. The change in https://github.com/WordPress/gutenberg/pull/63631/commits/f880412530c690e0e0f4877d6d59a9564cce1dfa bumps the specificity of the `a` rule to `0-1-1`.

---

To save another feedback cycle and review, I've tweaked the selector in https://github.com/WordPress/gutenberg/pull/63631/commits/95c4f7baba0b5506cea5f8bc5d5703a676eaaa23